torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1245,7 +1245,7 @@ at::Tensor XLANativeFunctions::div(
     c10::optional<c10::string_view> rounding_mode) {
   XLA_FN_COUNTER("xla::");
   at::ScalarType dtype = at::result_type(self, other);
-  auto operands = GetBinaryOperands(self, other);
+  auto operands = GetBinaryOperands(self, UnwrapNumber(other, dtype));
```

Comment:
Ah.. nice catch. All of the binary ops should have this `UnwrapNumber` logic baked in. However `div` now we have `rounding_mode` in its signature, we don't call `DoBinaryOp`.